### ISSUE TITLE
Text Input on Web does not work with Null Safety

### ISSUE THREAD

**Issue by creativecreatorormaybenot** [flutter/flutter#62724]
## Steps to Reproduce

0. `flutter run --enable-experiment=non-nullable --no-sound-null-safety -d chrome`.
1. Create a `TextField`.
2. Focus on the `TextField`.
3. Keyboard input is ignored.

This is the case for ***all* text fields** since I upgraded to the latest `master` version.

```
Flutter 1.21.0-6.0.pre.141 • channel master • https://github.com/flutter/flutter.git
Framework • revision c748f1a7aa (10 hours ago) • 2020-07-31 17:45:18 -0700
Engine • revision 280bbfc763
Tools • Dart 2.10.0 (build 2.10.0-2.0.dev bd528bfbd6)
```

It worked fine for `Flutter 1.21.0-6.0.pre.138`. That is *all* text fields worked fine.

**Comment by creativecreatorormaybenot** [comment]
If I had to guess what caused it, I would say https://github.com/flutter/flutter/commit/7884420a0a32467fa268761b99cf19c58480a499 by @a14n because text input for null safety is covered by that change.

The same app **does work** without the null safety flags:

```dart
import 'package:flutter/material.dart';

void main() {
  runApp(MaterialApp(home: Scaffold(body: TextField())));
}
```

So this *works* **without null safety** but does not work with null safety.

**Comment by creativecreatorormaybenot** [comment]
I was able to confirm that #62513 is the cause for this regression by using `git checkout 7cbec567de997daeb12837c362ee2a3940ef68de` (the previous commit). The issue is not present with that commit.

**Comment by a14n** [comment]
I can reproduce the issue in chrome but not on mobile. Additionally there are no error in the console to understand what happens :-/

**Comment by creativecreatorormaybenot** [comment]
@a14n I am experiencing the same behavior 😕 

This is also why it took me some time to nail it down to the commit, however, I was able to confirm that it has to be that commit and it does make sense because the text input service was touched for apps running with null safety.

**Comment by pedromassangocode** [comment]
<details>
<summary> Complete sample code</summary>

```
import 'package:flutter/material.dart';

void main() {
  runApp(MyApp());
}
class MyApp extends StatelessWidget {
  @override
  Widget build(BuildContext context) {
    return MaterialApp(
      title: 'Flutter Demo',
      home: MyHomePage(),
    );
  }
}
class MyHomePage extends StatelessWidget {
  @override
  Widget build(BuildContext context) {
    return Scaffold(
      body: Center(
        child: TextField(),
      ),
    );
  }
}
```
</details>

<details>
<summary> flutter doctor -v</summary>

```
[✓] Flutter (Channel master, 1.21.0-6.0.pre.142, on Mac OS X 10.15.6 19G73, locale en)
    • Flutter version 1.21.0-6.0.pre.142 at /Users/pedromassango/dev/SDKs/flutter_master
    • Framework revision d261f1ef58 (4 hours ago), 2020-08-03 00:59:36 -0400
    • Engine revision 083282e33b
    • Dart version 2.10.0 (build 2.10.0-2.0.dev 365525432a)

[!] Android toolchain - develop for Android devices (Android SDK version 30.0.1)
    • Android SDK at /Users/pedromassango/Library/Android/sdk
    • Platform android-30, build-tools 30.0.1
    • Java binary at: /Applications/Android Studio.app/Contents/jre/jdk/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 1.8.0_242-release-1644-b3-6222593)
    ! Some Android licenses not accepted.  To resolve this, run: flutter doctor --android-licenses

[✓] Xcode - develop for iOS and macOS (Xcode 11.6)
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Xcode 11.6, Build version 11E708
    • CocoaPods version 1.9.3

[✓] Chrome - develop for the web
    • Chrome at /Applications/Google Chrome.app/Contents/MacOS/Google Chrome

[!] Android Studio (version 4.0)
    • Android Studio at /Applications/Android Studio.app/Contents
    ✗ Flutter plugin not installed; this adds Flutter specific functionality.
    ✗ Dart plugin not installed; this adds Dart specific functionality.
    • Java version OpenJDK Runtime Environment (build 1.8.0_242-release-1644-b3-6222593)

[✓] IntelliJ IDEA Community Edition (version 2020.2)
    • IntelliJ at /Applications/IntelliJ IDEA CE.app
    • Flutter plugin version 47.1.4
    • Dart plugin version 202.6397.47

[✓] Connected device (3 available)
    • sdk gphone x86 (mobile) • emulator-5554 • android-x86    • Android 11 (API 30) (emulator)
    • Web Server (web)        • web-server    • web-javascript • Flutter Tools
    • Chrome (web)            • chrome        • web-javascript • Google Chrome 84.0.4147.105

! Doctor found issues in 2 categories.
Process finished with exit code 0
```
</details>

**Comment by a14n** [comment]
This should be fixed with #62778
